pkgs/shells/zsh/zsh-history/default.nix

```diff
@@ -0,0 +1,32 @@
+{ stdenv, lib, fetchFromGitHub, buildGoPackage }:
+
+with lib;
```

Comment:
Please don't `with` for the entire file, that make it hard to understand where things are coming from.

Reply:
Done